.devcontainer/devcontainer.json

```diff
@@ -1,28 +1,16 @@
 {
-	"name": "sense-base ROS2 with cuda-12.5.0-devel-ubuntu22.04",
+	"name": "base ROS2 with cuda-12.6.0-devel-ubuntu24.04",
```

Comment:
the name doesn't have any particular ramifications, but for correctness, its Cuda 12.1 with ubuntu 22.04.

Reply:
sorted out here [8f91458](https://github.com/sense-base/base/pull/38/commits/8f914581170cff80ae67d45fc9b3eddbf18be6e2)